lib/libutee/include/tee_api.h

```diff
@@ -326,6 +326,8 @@ void TEE_GenerateRandom(void *randomBuffer, size_t randomBufferLen);
 
 void TEE_GetSystemTime(TEE_Time *time);
 
+void TEE_GetSystemCounter(uint64_t *cnt);
```

Comment:
This function is not in the GP specs. Therefore, it should be in a separate file (probably `tee_internal_api_extensions.h`).